fe-games/src/components/Review.jsx

```diff
@@ -5,36 +5,36 @@ import { useParams } from "react-router-dom";
 
 export function Review() {
   const [currentReview, setCurrentReview] = useState([]);
-  const [isLoading, setLoading] = useState(true)
+  const [isLoading, setLoading] = useState(true);
 
   const { review_id } = useParams();
 
-  console.log(review_id);
-
   useEffect(() => {
     fetchReviewById(review_id).then(({ review }) => {
-        setLoading(false)
+      setLoading(false);
       setCurrentReview(review);
     });
   }, [review_id]);
 
-  if (isLoading){
-    return <span className="loader"></span>
+  if (isLoading) {
+    return <span className="loader"></span>;
   }
```

Comment:
I can see you have implemented an isLoading pattern, however when the articles are still loading, nothing shows on my screen, it's just blank apart from the header and nav bar. There's no indication to me, as a user, that data is loading? Maybe add a `<p>` tag with some text also?